output.md

```diff
@@ -90,59 +95,73 @@ file.h5
 # V0h and V1h are discretized SymPDE Space
 # u0 and u1 are FemFields belonging to either of those spaces
 output_m = OutputManager('spaces.yml', 'fields.h5')
 
-output_m.add_spaces(V0=V0h, V1=V1h) 
+output_m.add_spaces(V0=V0h, V1=V1h)
+output_m.export_spaces_info() # Writes the space information to Yaml
 
 output_m.set_static() # Tells the object to save in /static/
 output_m.export_fields(u0_static=u0, u1_static=u1) # Actually does the saving
 
-output_m.add_snapshot(t=0., ts=0) 
+output_m.add_snapshot(t=0., ts=0)
 # The line above tells the object to:
 # 1. create the group snapshot_x with attribute t and ts
 # 2. save in this snapshot
 output_m.export_fields(u0=u0, u1=u1)
-
-output_m.export_spaces_info() # Writes the space information to Yaml
 ```
 
 ## Usage of class `PostProcessManager`
 
 Typically the `PostProcessManager` class is used in a separate post-processing script, which is run after the simulation has finished. In essence it evaluates the FEM fields over a uniform grid (applying the appropriate push-forward operations) and exports the values to a VTK file (or a sequence of files in the case of a time series). An instance of the `PostProcessManager` class is created by specifying the following:
 
-1.  The name of the geometry file (in HDF5 format) which defines the multi-patch geometry of interest
+1.  The name of the geometry file or the topological domain
 2.  The name of the YAML file that contains the information about the FEM spaces
 3.  The name of the HDF5 file that contains the coefficients of all the FEM fields
 
-In order to export the fields to a VTK file, the user needs to prepare the evaluation grid `grid` and then call the method `export_to_vtk(base_name, grid, npts_per_cell, snapshots, fields)`, where `base_name` is the base name for the VTK output files, `npts_per_cell` specifies the refinement in the case of a uniform grid, `snapshots` specifies which time snapshots should be extracted from the HDF5 file (`None` in the case of static fields) and `fields` is a dictionary of `(vtk_field_name, h5_field_name)` pairs. Here is a usage example:
+In order to export the fields to a VTK file, the user needs to call the method `export_to_vtk(base_name, grid, npts_per_cell, snapshots, fields)`, where:
+1.  `base_name` is the base name for the VTK output files.
+2.  `grid` is either a user specified evaluation grid or `None`.
+3.  `npts_per_cell` specifies the refinement in the case of a uniform grid.
+4.  `snapshots` specifies which time snapshots should be extracted from the HDF5 file (`none` in the case of static fields)
+5.  `fields` is a tuple  of `h5_field_name`.
+
+Here is a usage example:
 
 ```python
 # geometry.h5 is where the domain comes from. See PostProcessManager's docstring for me information
 post = PostProcessManager(geometry_file='geometry.h5', space_file='spaces.yml', fields_file='fields.h5')
 
 # See PostProcessManager.export_to_vtk's and TensorFemSpace.eval_fields' doscstrings for more information
```

Comment:
Typo: doscstrings -> docstrings

Reply:
Fixed in [34bfd98](https://github.com/pyccel/psydac/pull/234/commits/34bfd9845a7a7c44d8589ff9b670b919f204d457)